src/components/UserPortal/StartPostModal/StartPostModal.tsx

```
@@ -233,11 +174,14 @@
             value={postContent}
           />
-          {img && (
-            <div className={styles.previewImage}>
-              <Image src={img} alt="Post Image Preview" />
-            </div>
-          )}
+          {attachment &&
+            attachment
+              .filter((att) => att.mimeType.startsWith('image/'))
+              .map((att) => (
+                <div className={styles.previewImage}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)